ddtrace/contrib/langchain/patch.py

```diff
@@ -675,6 +675,72 @@ async def traced_chain_acall(langchain, pin, func, instance, args, kwargs):
     return final_outputs
 
 
+@with_traced_module
+def traced_lcecl_runnable_sequence(langchain, pin, func, instance, args, kwargs):
+    integration = langchain._datadog_integration
+    span = integration.trace(pin, "%s.%s" % (instance.__module__, instance.__class__.__name__), interface_type="chain")
+    log_inputs = {}
+    log_outputs = {}
+    try:
+        inputs = get_argument_value(args, kwargs, 0, "input")
+        if integration.is_pc_sampled_span(span):
+            log_inputs = integration.tag_lcel_inputs(span, inputs)
```

Comment:
```suggestion
            integration.tag_lcel_inputs(span, inputs)
```
If we end up needing this actual value we can rename to something like `chain_inputs` instead of `log_inputs`, but since we're not using the value (because we're not logging) I think we can remove the actual variable itself.